docs/CTCORECHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## CleverTap Android SDK CHANGE LOG
 
+### Version 7.0.2 (October 10, 2024)
+
+#### New Features
+* Adds support for custom handshake domain configuration in android manifest
```

Comment:
Should we also add changelog related to json definition of custom templates?

Reply:
I think we should

Reply:
According to @vasct and @nzagorchev it will not be used by native SDK.
Will confirm with them once again.